addons/account/migrations/12.0.1.1/post-migration.py

```diff
@@ -0,0 +1,229 @@
+    )
+    openupgrade.logged_query(
+        cr, """
+        UPDATE res_company rc
+        SET transfer_account_code_prefix = act.transfer_account_code_prefix
+        FROM account_chart_template act
+        WHERE act.%s = rc.id
+        """, (
+            AsIs(openupgrade.get_legacy_name('company_id')),
+        ),
+    )
+
+
+def update_res_company_account_helper_states(cr):
+    openupgrade.logged_query(
+        cr, """
+        UPDATE res_company
+        SET account_dashboard_onboarding_state = 'done',
+            account_invoice_onboarding_state = 'done,
+            account_onboarding_invoice_layout_state = 'done',
+            account_onboarding_sale_tax_state = 'done',
+            account_onboarding_sample_invoice_state = 'done',
+        """
+    )
+
+
+def update_res_company_account_setup_steps_states(cr):
+    openupgrade.logged_query(
+        cr, """
+        UPDATE res_company
+        SET account_setup_bank_data_state = 'done'
+        WHERE account_setup_bank_data_done = TRUE
+        """
+    )
+    openupgrade.logged_query(
+        cr, """
+        UPDATE res_company
+        SET account_setup_fy_data_state = 'done'
+        WHERE account_setup_fy_data_done = TRUE
+        """
+    )
+    openupgrade.logged_query(
+        cr, """
+        UPDATE res_company
+        SET account_setup_coa_state = 'done'
+        WHERE account_setup_coa_done = TRUE
+        """
+    )
+
+
+def fill_account_journal_alias_id(env):
+    journal_model = env["account.journal"]
+    journals = journal_model.with_context(
+        active_test=False).search([('type', '=', 'purchase')])
+    for journal in journals:
+        journal._update_mail_alias({})
+
+
+def fill_account_move_reverse_entry_id(env):
+    all_moves = env['account.move'].search([])
+    if openupgrade.table_exists(env.cr, 'account_move_reverse'):
+        reversal_moves = all_moves.filtered(lambda m: m.reverse_entry_id). \
+            mapped('reverse_entry_id')
+        for move in reversal_moves:
+            move.reverse_date = 2999
+    else:
+        installed_langs = env['res.lang'].search([])  # search only active
+        for lang in installed_langs:
+            reversal_text = env['ir.translation']._get_source(
+                name=False, types='code', lang=lang.code,
+                source='reversal of: ')
+            reversal_moves = all_moves.filtered(
+                lambda m: m.ref and m.ref.startswith(reversal_text))
+            for move in reversal_moves:
+                name = move.ref.partition(reversal_text)[2]
+                origin = all_moves.filtered(lambda m: m.name == name)[0]
+                if not origin.reverse_entry_id:
+                    origin.reverse_entry_id = move.id
+                move.reverse_date = 2999
+    env['ir.filters'].create({
+        'name': 'To be reversed (undetermined date)',
+        'model_id': 'account.move',
+        'domain': [('auto_reverse', '=', True), ('reverse_date', '=', 2999)],
```

Comment:
Adjust the filter properly and don't add it `account_reversal` was not installed